pulse2percept/utils.py

```diff
@@ -113,6 +130,47 @@ def __init__(self, tsample, data):
     def __getitem__(self, y):
         return TimeSeries(self.tsample, self.data[y])
 
+    def __add__(self, other):

```

Comment:
This is confusing, because I would expect an addition of the underlying arrays (element-by-element). Can we implement this as another method, not overloading `add`?